openstack_controller/tests/conftest.py

```diff
@@ -138,6 +138,10 @@ def mock_responses(microversion_headers):
 
     def process_files(dir, response_parent):
         for file in dir.rglob('*'):
+            if 'clusters' in str(file.relative_to(dir)):
```

Comment:
Did you use this for debugging? Probably you should remove it before merging

Reply:
oops haha